prow/config.yaml

```diff
@@ -303,6 +302,30 @@ presubmits:
           - "--cov-threshold-percentage=0"
           - "--github-token=/etc/github-token/oauth"
 
+  - name: pull-tekton-pipeline-check-testgrid-config
+    run_if_changed: '^(prow/.*\.yaml)|(testgrid\.yaml)$'
```

Comment:
This job is located under "presubmit", meaning it will run before every PR. You probably want it under "postsubmit", where it will run (and update) after a prow or testgrid file is changed.

Reply:
The general pattern is to run this image as a presubmit with `--validate-config-file` (to make sure the updates will work) and also run this image as a postsubmit with `--output`

Reply:
got it thanks! I removed "preset-presubmit-sh" from the postsubmit job since it seems to be for presubmits (none of the postsubmits have it) but I think you mentioned that was needed for credentials-- is this OK?

Reply:
I'm guessing "preset-postsubmit-sh" might be the equivalent for setting up credentials in a postsubmit: https://github.com/tektoncd/plumbing/blob/0c4e9f3eb7143c105fe41df622f72fda6bed3d1e/prow/config.yaml#L106-L120

Reply:
nice, thanks!